addons/binding/org.openhab.binding.foobot/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="airboxlab"
```

Comment:
hmmm. Is this meant to be the `airboxlab` binding or the `foobot` binding? IMHO the naming should be consistent to avoid confusion.